packages/discord.js/src/structures/Message.js

```diff
@@ -645,12 +645,11 @@ class Message extends Base {
    * channel.bulkDelete(messages.filter(message => message.bulkDeletable));
    */
   get bulkDeletable() {
-    const permissions = this.channel?.permissionsFor(this.client.user);
     return (
       (this.inGuild() &&
         Date.now() - this.createdTimestamp < MaxBulkDeletableMessageAge &&
         this.deletable &&
-        permissions?.has(PermissionFlagsBits.ManageMessages, false)) ??
+        this.channel?.permissionsFor(this.client.user).has(PermissionFlagsBits.ManageMessages, false)) ??
```

Comment:
```suggestion
        this.channel?.permissionsFor(this.client.user)?.has(PermissionFlagsBits.ManageMessages, false)) ??
```

as members.me doesn't have to be cached everytime

Reply:
The `this.deletable` check above already handles this.